juju/juju
# Lxd net device logging

## Description of change

Since the post-LXD-refactor network checking has been recruited in the provider, we see the log entry for successful detection of NIC device config too frequently in the logs.

This notification is not particularly valuable for information or diagnostics, so we don't need to see it by default, particularly on every instantiation of LXD in the provider.

Change applied in 2.4 for later rolling into develop.

## QA steps

Bootstrap and add some machines. Tail the debug log and ensure that the logs indicate the changed level.

## Documentation changes

None.

## Bug reference

N/A
